rules/do-you-regularly-inspect-and-adapt-at-scale/rule.md

```diff
@@ -1,39 +1,50 @@
 ---
 type: rule
 title: Do you regularly inspect and adapt at scale?
-uri: do-you-regularly-inspect-and-adapt-at-scale
+uri: regularly-inspect-and-adapt-at-scale
```

Comment:
need to update folder name to match uri

Reply:
Done ✅ - PR for folder changes: https://github.com/SSWConsulting/SSW.Rules.Content/pull/8357